### PR TITLE
RHELPLAN-44234 - Logging - travis/molecule needs to be integrated with linux-system-roles template

### PR DESCRIPTION
**Pull request by nhosoi**
Synchronized with linux-system-roles/template using sync-template.sh

cmdline:
  ./sync-template.sh --workdir ~/linux-system-roles --local --preserve --repolist logging --force-copy

- Adding E204 (Lines should be no longer than 160 chars) to skip_list in .ansible-lint
- Disabling line-length in .yamllint.yml
- Removing molecule/default/tests/test_default.py.
- Omitting centos-6 from the platforms in molecule.yml.

### CODE REVIEW

**Comment by richm** [comment]
This is not exactly the same as what is in the template.  If you want what is in the template, use https://github.com/linux-system-roles/auto-maintenance/blob/master/sync-template.sh
* assumes you have a local working directory for lsr with subdirectories for roles e.g. `~/linux-system-roles/template ~/linux-system-roles/auto-maintenance ~/linux-system-roles/logging` etc. Note that the name `~/linux-system-roles` is not important - what is important is that all of the template, role, etc. are children of the same parent directory
* git clone and git update the latest linux-system-roles/template master branch
* in ~/linux-system-roles/logging - git commit/stash your unsaved changes - git checkout master

Then
```
cd ~/linux-system-roles/auto-maintenance
./sync-template.sh --workdir ~/linux-system-roles --local --preserve --repolist logging --force-copy
```
This will copy the latest changes from template, including files from the template that you may have changed locally (so please review them), will create a branch for these changes like `lsr-template-sync`, and will stage the changes (e.g. will do a `git add` for all of these changes - so use `git diff --cached` to see the changes).

**Comment by nhosoi** [comment]
Thank you, @richm! The sync-template script is very nice.

So, once https://github.com/linux-system-roles/template/pull/11 is merged, I can repeat this again, right?

**Comment by nhosoi** [comment]
Hmm, it was too early to have removed [WIP]... :) Investigating.

**Comment by richm** [comment]
> Thank you, @richm! The sync-template script is very nice.
> 
> So, once [linux-system-roles/template#11](https://github.com/linux-system-roles/template/pull/11) is merged, I can repeat this again, right?

It depends.  Is it that you want to use molecule v3?  If so, then we likely won't support for a while.  Is it that you want to sync with the latest template?  If so, then you don't have to wait.

**Comment by nhosoi** [comment]
Note: This pr depends on https://github.com/linux-system-roles/logging/pull/139 and https://github.com/linux-system-roles/logging/pull/140.

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -6,20 +7,32 @@ driver:
...
   options:
-    config-file: molecule/default/yaml-lint.yml
+    config-file: .yamllint.yml
 platforms:
-  - name: instance
-    image: ${MOLECULE_DISTRO:-"centos:7"}
-    command: /sbin/init
+  - name: centos-6
+    image: docker.io/linuxsystemroles/centos-6
+    privileged: true
```
Comment (richm) [inline molecule/default/molecule.yml:14]: You should omit `centos-6` if the role isn't supported on centos-6

**Comment by nhosoi** [comment]
@richm, newly added files are under MIT license - # SPDX-License-Identifier: MIT
Do I need to add MIT to the LICENSE file, as well?

**Comment by richm** [comment]
> @richm, newly added files are under MIT license - # SPDX-License-Identifier: MIT
> Do I need to add MIT to the LICENSE file, as well?

Yes

**Comment by nhosoi** [comment]
> > @richm, newly added files are under MIT license - # SPDX-License-Identifier: MIT
> > Do I need to add MIT to the LICENSE file, as well?
> 
> Yes

@richm, I submitted https://github.com/linux-system-roles/logging/pull/142.

**Comment by nhosoi** [comment]
Closing this PR in favor of the work across the roles by @richm.

**Pull request closed without merging**